modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -821,6 +827,7 @@ public CompletableFuture<Ignite> start(Path configPath) {
                         // Start all other components after the join request has completed and the node has been validated.
                         try {
                             lifecycleManager.startComponents(
+                                    threadPools,
```

Comment:
True for now, but when we will expand thread switching logic to cover meta storage it'll be necessary to move it to the "first" components group.